tests/gpu-container

```diff
@@ -130,6 +130,19 @@ if hasNeededAPIExtension "gpu_cdi"; then
     lxc start c1
     [ "$(lxc exec c1 -- ls /dev/dri/ | grep -c '^card[0-9]')" = "1" ]
     lxc exec c1 -- nvidia-smi
+
+    echo "==> Testing running nested Docker container with nvidia runtime"
+
+    lxc stop --force c1
+
+    lxc config set c1 security.nesting=true nvidia.driver.capabilities=all
+    lxc start c1
+
+    lxc exec c1 -- nvidia-smi
+
+    lxc exec c1 -- snap install docker
```

Comment:
I guess that by now the ugly `snapd` workaround we've been carrying for a while is no longer needed. We'll see how it goes to be certain but I think it's behind us now :)